packages/api-core/src/domain-suggestions/types.ts

```diff
@@ -181,6 +183,17 @@ export interface DomainSuggestion {
 	 * Whether the client should show the dot gay notice
 	 */
 	dot_gay_notice_required?: true;
+
+	/**
+	 * Policy notices associated with the domain
+	 */
+	policy_notices?: PolicyNotice[];
+}
+
+export interface PolicyNotice {
+	type: string;
+	label: string;
+	message: TranslateResult;
```

Comment:
This is a computed field, right? So it's not returned from the API itself. We shouldn't add this here since we're not defining it here.

Reply:
Oh if it's there and it's a string, then why are we saying it's `TranslateResult`? We should keep it as a string, right?

Reply:
Yeah, good question. I was a bit forced by TypeScript to do it. 😅 The reason is that I'm modifying the message in the `getPolicyNoticeMessage` function. More context about why I'm doing it can be found in [this comment](https://github.com/Automattic/wp-calypso/pull/105532/files#r2319473110).